services/index.js

```diff
@@ -3,38 +3,38 @@ import doContactUs from './contact_us';
 import doSignUp from './mailchimp/sign-up/index';
 import doUpdateUser from './mailchimp/update-user/index';
 
-export function publish(event, context, cb) {
-  if (event.query && event.query.auth_token !== process.env.PRIVATE_LAMBDA_API_KEY) {
-    return cb(new Error('[403] Forbidden'));
+const cbWithErrorHandling = cb => (e, body) => {
+  if (e) {
+    e.message = `[500] ${e.message}`; // eslint-disable-line no-param-reassign
+    return cb(e);
   }
+  return cb(null, body);
+};
 
+const errorHandlerWrapper = (f, event, context, cb) => {
   try {
-    doPublish(cb);
+    f(event, context, cbWithErrorHandling(cb));
   } catch (e) {
-    cb(e);
+    cbWithErrorHandling(cb)(e);
   }
+};
```

Comment:
I think it'd be best to curry this function like we do with `cbWithErrorHandling`. Secondly, we could probably inline the `cbWithErrorHandling` stuff now it isn't repeated.